wordpress-mobile/WordPress-iOS
# Comment Moderation Bar: add initial view

Ref: #17200

This adds an initial `CommentModerationBar` view with the `Pending` button. Tapping the button will toggle the UI to indicate selected or not. No action is actually performed yet.

NOTE: The bar is _temporarily_ displayed in `CommentContentTableViewCell`. Since there are [pending](https://github.com/wordpress-mobile/WordPress-iOS/pull/17147) changes to this cell, I will undo the changes here before merging and add it back later. (Hence the `DO NOT MERGE` label.)

To test:
- Enable the `newCommentDetail` feature.
- Go to My Site > Comments > Comment details.
- Verify the bar is displayed in the `CommentContentTableViewCell`, with the `Pending` button in the unselected state.
- Tap the `Pending` button. Verify the UI changes to the selected state.


| Unselected | Selected |
|--------|-------|
| <img width="445" alt="light_unselected" src="https://user-images.githubusercontent.com/1816888/134562496-c6744777-421d-4405-a71b-54aab9f3ea5e.png"> | <img width="441" alt="light_selected" src="https://user-images.githubusercontent.com/1816888/134562556-d44ef507-5c0b-4dab-bdbe-c2198446ac17.png"> |
| <img width="442" alt="dark_unselected" src="https://user-images.githubusercontent.com/1816888/134562598-c36a7c40-24c8-4bd7-b46d-62863e4f508b.png"> | <img width="448" alt="dark_selected" src="https://user-images.githubusercontent.com/1816888/134562626-d692bc20-408a-485a-9460-758e00b16773.png"> |


## Regression Notes
1. Potential unintended areas of impact
N/A. Feature is incomplete and disabled.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A. Feature is incomplete and disabled.

3. What automated tests I added (or what prevented me from doing so)
N/A. Feature is incomplete and disabled.

PR submission checklist:
